package-lock.json

```diff
@@ -20,10 +20,10 @@
       "dev": true,
       "requires": {
         "@babel/types": "7.0.0-beta.44",
-        "jsesc": "^2.5.1",
-        "lodash": "^4.2.0",
-        "source-map": "^0.5.0",
-        "trim-right": "^1.0.1"
+        "jsesc": "2.5.1",
```

Comment:
This file appears to have changes unrelated to this PR -- we usually coordinate our dependency upgrades once per release. Can you revert?

Reply:
removed the package-lock.json from the commit